tools/rimage/src/manifest.c

```diff
@@ -512,6 +512,8 @@ static int man_module_create_reloc(struct image *image, struct manifest_module *
 		unsigned int j;
 
 		strncpy(name, (char *)sof_mod->module.name, SOF_MAN_MOD_NAME_LEN);
```

Comment:
s/strncpy/strlcpy ?

Reply:
This would be a further update if needed, I think we are good to go today with @abonislawski fix.